Formula/tweak.rb

```diff
@@ -10,6 +10,6 @@ def install
   end
 
   test do
-    assert_equal "# Default .tweakrc generated by `tweak -D'.\n", `tweak -D | head -1`
+    assert_equal "# Default .tweakrc generated by `tweak -D'.", `tweak -D`.lines.first.strip
```

Comment:
tried using tweak -D which outputs the default .tweakrc file to stdout ...
alternatively could ... maybe write that to a file? then using tweak to read it? but then couldn't figure out how to compare the contents of an emacs window with asserts